packages/example/package.json

```diff
@@ -15,7 +15,7 @@
   },
   "devDependencies": {
     "cross-env": "6.0.3",
-    "cypress-example-kitchensink": "2.0.2",
+    "cypress-example-kitchensink": "2.0.3",
```

Comment:
`cypress-example-kitchensink` has been bumped without running `yarn`, so now `yarn.lock` is out of sync still using the previous version `"cypress-example-kitchensink": "2.0.2"`.

Reply:
- This issue has been sorted out by https://github.com/cypress-io/cypress/pull/29128 which updated `yarn.lock`